notebooks/03-query/05-lookup-essentials.ipynb

```diff
@@ -0,0 +1,1561 @@
+    "\"ROC\": \"massive_airport\",\n",
+    "\"RST\": \"massive_airport\",\n",
+    "\"RSW\": \"massive_airport\",\n",
+    "\"SAN\": \"massive_airport\",\n",
+    "\"SAT\": \"massive_airport\",\n",
+    "\"SAV\": \"massive_airport\",\n",
+    "\"SBA\": \"medium_airport\",\n",
+    "\"SBN\": \"massive_airport\",\n",
+    "\"SBP\": \"medium_airport\",\n",
+    "\"SCC\": \"medium_airport\",\n",
+    "\"UNV\": \"medium_airport\",\n",
+    "\"SDF\": \"massive_airport\",\n",
+    "\"SEA\": \"massive_airport\",\n",
+    "\"SFO\": \"massive_airport\",\n",
+    "\"SGF\": \"massive_airport\",\n",
+    "\"SGU\": \"medium_airport\",\n",
+    "\"SHV\": \"medium_airport\",\n",
+    "\"SIT\": \"medium_airport\",\n",
+    "\"SJC\": \"massive_airport\",\n",
+    "\"SJT\": \"medium_airport\",\n",
+    "\"SJU\": \"massive_airport\",\n",
+    "\"SLC\": \"massive_airport\",\n",
+    "\"SMF\": \"massive_airport\",\n",
+    "\"SMX\": \"medium_airport\",\n",
+    "\"SNA\": \"massive_airport\",\n",
+    "\"SPI\": \"massive_airport\",\n",
+    "\"SPS\": \"massive_airport\",\n",
+    "\"SRQ\": \"massive_airport\",\n",
+    "\"STL\": \"massive_airport\",\n",
+    "\"STT\": \"medium_airport\",\n",
+    "\"STX\": \"medium_airport\",\n",
+    "\"SUN\": \"medium_airport\",\n",
+    "\"SWF\": \"medium_airport\",\n",
+    "\"SYR\": \"massive_airport\",\n",
+    "\"TLH\": \"massive_airport\",\n",
+    "\"TOL\": \"massive_airport\",\n",
+    "\"TPA\": \"massive_airport\",\n",
+    "\"TRI\": \"massive_airport\",\n",
+    "\"TUL\": \"massive_airport\",\n",
+    "\"TUP\": \"medium_airport\",\n",
+    "\"TUS\": \"massive_airport\",\n",
+    "\"TVC\": \"medium_airport\",\n",
+    "\"TWF\": \"medium_airport\",\n",
+    "\"TXK\": \"medium_airport\",\n",
+    "\"TYR\": \"medium_airport\",\n",
+    "\"TYS\": \"massive_airport\",\n",
+    "\"VLD\": \"medium_airport\",\n",
+    "\"VPS\": \"massive_airport\",\n",
+    "\"WRG\": \"medium_airport\",\n",
+    "\"XNA\": \"medium_airport\",\n",
+    "\"YAK\": \"medium_airport\",\n",
+    "\"NYL\": \"medium_airport\"\n",
+    "                    }\n",
+    "            }\n",
+    "        }\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4eb55f18-c356-4f89-b938-c4b6a72391e6",
+   "metadata": {},
+   "source": [
+    "Use the next cell to post this new definition to the API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ae669896-2dd3-4129-9fc1-5b848fce94f7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "postLookup(lookup_definition)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c736d0e9-d90c-47ae-90f4-7f2c78ee06b2",
+   "metadata": {},
+   "source": [
+    "Now run the next cell to keep an eye on the distribution process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d73e31a-0d1b-4750-8e9c-258d67572698",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "waitForLookup(lookup_tier, lookup_name, 60)"
```

Comment:
I found the issue here – it's because the default period to wait before doing a lookup update check is 2 minutes. I've updated the waitForLookup throughout to 2 minutes, and have added a comment in the function def to clarify why this is set to that.